Backpack-SwiftUI/Calendar/Classes/Core/CalendarTypeContainerFactory.swift

```diff
@@ -49,97 +39,64 @@ struct CalendarTypeContainerFactory<MonthHeader: View, DayAccessoryView: View>:
             validRange: validRange,
             monthScroll: monthScroll
         ) { month in
-            monthHeader(month)
-            CalendarMonthGrid(
-                monthDate: month,
-                calendar: calendar,
-                validRange: validRange,
-                dayCell: returnMakeCellFunction(),
-                emptyLeadingDayCell: emptyLeadingDayCell,
-                emptyTrailingDayCell: emptyTrailingDayCell,
-                dayAccessoryView: dayAccessoryView
-            )
-        }
-    }
-    
-    func returnMakeCellFunction() -> ((Date) -> CalendarSelectableCell) {
-        return { dayDate in
-            CalendarSelectableCell(
-                selectionType: selectionType,
-                calendar: calendar,
-                accessibilityProvider: calendarAccessibilityConfiguration,
-                dayDate: dayDate,
-                onSelection: handleSelection
-            )
-        }
-        
-    }
-    
-    @ViewBuilder func emptyLeadingDayCell(for emptyDayInfo: EmptyCellInfo) -> some View {
-        switch selectionType {
```

Comment:
most this changes are reverting back to the old implementation. with an exception mentioned below